packages/pluggableWidgets/accordion-web/src/components/Accordion.tsx

```diff
@@ -21,11 +35,45 @@ export interface AccordionProps extends Pick<AccordionContainerProps, "class" |
 export function Accordion(props: AccordionProps): ReactElement | null {
     const reducer = useRef(getCollapsedAccordionGroupsReducer(props.singleExpandedGroup ? "single" : "multiple")); // the accordion group reducer function doesn't need to change during the lifetime of this component, since the singleExpandedGroup won't change.
 
-    const [collapsedAccordionGroups, collapsedAccordionGroupsDispatch] = useReducer(
+    const [accordionGroupCollapsedState, accordionGroupCollapsedStateDispatch] = useReducer(
         reducer.current,
-        props.groups.map(() => !props.previewMode && props.collapsible)
+        undefined,
```

Comment:
Should the init value be `undefined` or rather something like `[]`? I'm not entirely familiar with the API for `userReducer`

Reply:
It's the initialiser argument that the initialising function is called with. Since we `map` over `props.groups` inside that function we don't need an empty array.

Reply:
Ahhh, okay that makes sense 👍